src/iop/exposure.c

```diff
@@ -274,6 +274,24 @@ void init_presets (dt_iop_module_so_t *self)
                                                          .deflicker_target_level = -4.0f,
                                                          .compensate_exposure_bias = FALSE},
                              sizeof(dt_iop_exposure_params_t), 1);
+
+
+  if(strcmp(dt_conf_get_string("plugins/darkroom/workflow"), "scene-referred") == 0)
```

Comment:
I won't make this preset creating conditional this will avoid one case where we need to restart dt to swicth from scene-referred to another choice. Yes I know there is other case like that for basecurve for example.

Reply:
Of course if we do that we need to auto-apply only for the scene-referred workflow and we are back to the basecure case anyway :(